### DISCUSSION TITLE
Welcome to vscode-R Discussions!

### DISCUSSION THREAD

**Discussion by Ikuyadeu** [REditorSupport/vscode-R#590] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danielbasso** [comment]
Hey @Ikuyadeu , thanks for opening this tab! I'm just starting in this whole github/community development thing; this project is acting like a training, so I'm prone to make some mistakes. The current issues tab is a bit intimidating for newcomers, because it has an implied "criticism" on it. This tab is ideal for asking "neutral" questions.

I recon that it could became a little messy if people start to post issues in the discussion tab and vice-versa. Just to take note, at least, github provides a functionality to transform issues in discussions. You can even create a specific issue label that automatically do this reassign as stated [here](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository#converting-issues-based-on-labels). The opposite isn't available right now, but it shouldn't be a problem, just create a new issue and reference the discussion (maybe it's even better this way).

I think that, with proper care, this tab could act as a nice Q&A for newcomers, and at the same time clean the issues tab a bit. Thanks again.

**Comment by malcook** [comment]
I'm a long-time emacs/org-mode user considering what advice regarding IDEs/editors/platforms to give to new data scientist wannabes besides RStudio++  and thinking that this is the best new entrant into the field.  I 've spinned it up a few times and its beginning to feel quite nice.
